content/applications/sales/sales/send_quotations/get_paid_to_validate.rst

```diff
@@ -1,53 +1,68 @@
-============================
-Get paid to confirm an order
-============================
+=================================
+Online payment order confirmation
+=================================
 
-In general, online payments are considered as a fast and secure alternative to traditional payment
-methods. It is generally cheaper, easier and faster than other payment methods. It is particularly
-useful and efficient for international transactions. With Odoo Sales, you can use online payments
-to get automatic orders confirmation. Online payments are made instantly, so it's very convenient
-and saves lots of time in a basic sales process.
+With Odoo *Sales*, online payments can be used to get automatic confirmation on orders.
 
 Enable online payment
 =====================
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Online Payment**
-feature.
+Go to :menuselection:`Sales app --> Configuration --> Settings`, scroll to the
+:guilabel:`Quotations & Orders` heading, and activate the :guilabel:`Online Payment` feature, by
+checking the box next to it, and clicking :guilabel:`Save`.
 
-.. image:: get_paid_to_validate/get_paid_1.png
+.. image:: get_paid_to_validate/online-payment-setting.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to enable online payment on Odoo Sales?
+   :alt: How to enable online payment on Odoo Sales.
 
-There, you will have direct access to the **Payment Acquirers** page. It will allow you to select
-and configure your acquirers of choice. Before creating or modifying a payment acquirer, be sure to
-check out our documentation about how to be paid with payment acquirers such as
-:doc:`/applications/finance/payment_acquirers/paypal`,
-:doc:`/applications/finance/payment_acquirers/authorize`, and others in the
-:doc:`/applications/finance/payment_acquirers` documentation.
+After checking the box beside the :guilabel:`Online Payment` feature, a link to the
+:guilabel:`Payment Acquirers` appears beneath it. Clicking that link reveals the entire
+:guilabel:`Payment Acquirers` page.
```

Comment:
```suggestion
After checking the box beside the :guilabel:`Online Payment` feature, a link to the :guilabel:`Payment Acquirers` page appears beneath it.
```

Reply:
I understand cutting that last portion to be more succinct, but since it is necessary to proceed with the flow, I think I'm going to keep it in, just to avoid any confusion. But, again, I totally get where you're coming from with this suggestion.